### PR TITLE
Add TS + webpack examples

### PR DESCRIPTION
**Pull request by AndriyShevchuk**
Add example how to use SDK TS in web browser using typescript.
Webpack bundle all code and deploy it on dev server in browser.

Resolves: OLPEDGE-1720
Signed-off-by: ashevchu <ext-andrii.shevchuk@here.com>

### CODE REVIEW

File: docs/examples/ts/example.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  QuadKeyPartitionsRequest,
+  quadKeyFromMortonCode,
+  IndexLayerClient,
+  IndexQueryRequest,
+  StreamLayerClient,
+  SubscribeRequest,
+  PollRequest,
+  SeekRequest,
+  VolatileLayerClient
+} from "../../../@here/olp-sdk-dataservice-read";
```
Comment (OleksiiZubko) [inline docs/examples/ts/example.ts]: Please, use npm packages

File: docs/examples/ts/example.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+import {
+  UserAuth,
+  requestToken,
+  HttpError
+} from "../../../@here/olp-sdk-authentication/index.web";
```
Comment (OleksiiZubko) [inline docs/examples/ts/example.ts]: Please use npm packages and import from @here/olp-sdk-authentication

File: docs/examples/ts/example.ts (hunk 1 above)
Comment (AndriyShevchuk) [reply docs/examples/ts/example.ts]: Sure. I forgot to replace it. Done

File: docs/examples/ts/example.ts (hunk 2 above)
Comment (AndriyShevchuk) [reply docs/examples/ts/example.ts]: Done

File: docs/examples/ts/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const tsFiles = [
```
Comment (OleksiiZubko) [inline docs/examples/ts/webpack.config.js]: Why do we need that?

Comment (AndriyShevchuk) [reply docs/examples/ts/webpack.config.js]: To make sure that example.ts will be the last one in the list of files. Files path changed.

**Pull request merged**